plugins/rikki/heroeslounge/components/updatematch/default.htm

```diff
@@ -184,4 +193,47 @@ <h5 class="modal-title">New Game</h5>
     </div>
 </div>
 <!-- End New Game Modal -->
+
+<!-- Start Reschedule Match Modal -->
+<div class="modal fade" role="dialog" id="reschedule_{{__SELF__.match.id}}">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content card-body" id="schedulebox{{__SELF__.match.id}}">
+            <h4 class="card-title text-center">VS <a href="{{ 'team/view' | page({slug:__SELF__.opp.slug})}}"><strong>{{__SELF__.opp.title |upper}}</strong></a></h4>
+            <form>
+                <div class="row mb-3">
+                    <div class="col-md-8 col-8 text-center">
+                        <input type="text" class="match text-center" style="height:100%" name="date" value="{{__SELF__.match.wbp | date(__SELF__.datetimeFormat, __SELF__.timezone)}}">
+                        <input type="hidden" name="match_id" value="{{__SELF__.match.id}}">
+                        <input type="hidden" name="opponent_id" value="{{__SELF__.opp.id}}">
+                    </div>
+                </div>
+                <p class="card-text">Timezone: {{__SELF__.timezone}} {{__SELF__.timezoneOffset}}</p>
+                <p class="card-text">This match has to be played before: {{__SELF__.match.tbp | date(__SELF__.datetimeFormat, __SELF__.timezone)}}</p>
+                <p class="card-text"><small class="text-muted">If you fail to report the score for the game before the last play time, the match will be marked as a tie regardless of score.</small></p>
```

Comment:
This seems duplicated information with the "has to be reported by ..." further above. Try to unify this in an elegant way. (like "game needs to be played and its results reported by ...")